tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -3267,6 +3276,184 @@ void main()
   VERIFY_IS_TRUE(FoundTheVariable);
 }
 
+class DxcBlobImpl : public IDxcBlob {
```

Comment:
You can use blob creation methods of `IDxcUtils` instead of rolling your own blob type.

Reply:
Right you are. In fact, now that I think about it, I already have that in pixtest.